content/dstu2/allergy-intolerance.md

```diff
@@ -0,0 +1,128 @@
+---
+title: AllergyIntolerance | FHIR DSTU 2 API
+---
+
+# AllergyIntolerance
+
+* TOC
+{:toc}
+
+## Search
+
+Search for AllergyIntolerances that meet supplied query parameters:
+
+    GET /AllergyIntolerance?:parameters
+
+### Parameters
+
+ Name    | Required? | Type                                                           | Description
+---------|-----------|----------------------------------------------------------------|---------------------------------------------
+`patient`|     Y     |[`reference`](http://hl7.org/fhir/DSTU2/search.html#reference)| Who the sensitivity is for. Example: `12345`
+`status` |     N     |[`token`](http://hl7.org/fhir/DSTU2/search.html#token)        | [Certainty of the allergy or intolerance](http://hl7.org/fhir/DSTU2/valueset-allergy-intolerance-status.html). Example: `confirmed`
+
+### Response
+
+<%= headers 200 %>
+<%= json(:dstu2_allergy_intolerance_bundle) %>
+
+## Create
+
+Create new allergies.
+
+    POST /AllergyIntolerance
+
+### Headers
+
+To successfully POST an allergy, the following headers must be provided. Allergy creation is supported from the closed endpoint only.
+
+    Content-Type: application/json+fhir
+    Authorization: <OAuth2 Bearer Token>
+
+### Body fields
+
+<%= definition_table(:allergy_intolerance, :create, :dstu2) %>
+
+### Example Body
+
+<%= json(:dstu2_allergy_intolerance_create) %>
+
+### Response
+
+<%= headers 201 %>
+<pre class="terminal">
+    Connection → Keep-Alive
+    Content-Encoding → gzip
+    Content-Length → 20
+    Content-Type → text/html; charset=UTF-8
+    Date → Wed, 13 Jan 2016 21:45:47 GMT
+    Keep-Alive → timeout=15, max=100
+    Last-Modified → Tue, 15 Dec 2015 19:13:20 GMT
+    Status → 201 Created
+    access-control-allow-methods → DELETE, GET, POST, PUT, OPTIONS, HEAD
+    access-control-allow-origin → *
+    access-control-expose-headers → ETag, Content-Location, Location, X-Request-Id, WWW-Authenticate, Date
+    access-control-max-age → 0
+    cache-control → no-cache
+    etag → W/"35502191"
+    location → https://fhir.sandboxcernerpowerchart.com/dstu2/9e2aaa6d-3811-4d84-b5f9-93ccf529eefa/AllergyIntolerance/35502191
+    server-response-time → 1260.984596
+    strict-transport-security → max-age=631152000
+    vary → Origin,User-Agent,Accept-Encoding
+    x-content-type-options → nosniff
+    x-frame-options → SAMEORIGIN
+    x-request-id → edad5615-1f19-4ff0-9999-46f46a52368e
+    x-runtime → 1.260940
+    x-xss-protection → 1; mode=block
+</pre>
+
+The `etag` response header indicates the current `If-Match` version to use on subsequent updates.
+
+## Update
+
+Update an existing allergy.
```

Comment:
We should add a disclaimer about any field that is missing is interpreted as nulling out/removing that field? Link to information about this in the spec: http://hl7.org/fhir/DSTU2/http.html#update

Or, update the general doc page around how updates behave and link to it from resources allowing updates?